corehq/apps/app_manager/static/app_manager/js/case-knockout-bindings.js

```diff
@@ -101,9 +101,7 @@ ko.bindingHandlers.casePropertyTypeahead = {
 ko.bindingHandlers.accordion = {
     init: function(element, valueAccessor) {
         var options = valueAccessor() || {};
-        setTimeout(function() {
-            $(element).accordion(options);
-        }, 0);
```

Comment:
I didn't thoroughly dig through why all of them were necessary (in particular, swapping `load` and `save` in `case-config-ui-advanced.js` felt weird) but yeah, I did test the accordion.